Clarify sentry pagerduty alert config (#8401)
getsentry/sentry-docs: src/docs/product/integrations/notification-incidents/pagerduty/index.mdx

![PagerDuty metric alert rule action](pd_metric_alert_rule.png)

> If a PagerDuty incident is created from the critical trigger of a metric alert with no warning threshold set, it will resolve itself when the Sentry metric alert is resolved. If the metric alert has a warning threshold set, you may need to manually resolve the PagerDuty incident.

Reviewer: ```suggestion
If a PagerDuty incident is created from the critical trigger of a metric alert and no warning threshold has been set, it will resolve itself when the Sentry metric alert is resolved. Metric alerts that have set warning thresholds may need to be manually resolved as PagerDuty incidents.
```
